app/views/sessions/purchase_cart.html.erb

```diff
@@ -7,7 +7,11 @@
     </tr>
     <% if session[:cart] %>
         <% session[:cart].each do |item_id, quantity| %>
-        <% item = Item.find_by(id: item_id) %>
+            <% item = Item.find_by(id: item_id) %>
+            <!-- Line below generates receipt for purchasing this item, if possible -->
+            <% if logged_in? %>
```

Comment:
Should there be a URL? Would that be easier than having a separate tool?
On Mon, Jun 22, 2015 at 1:46 AM rynkwn <notifications@github.com> wrote:

> In app/views/sessions/purchase_cart.html.erb
> <https://github.com/Niupiao/niupiao-market/pull/80#discussion_r32906545>:
>
> > @@ -7,7 +7,11 @@
> >      </tr>
> >      <% if session[:cart] %>
> >          <% session[:cart].each do |item_id, quantity| %>
> > -        <% item = Item.find_by(id: item_id) %>
> > +            <% item = Item.find_by(id: item_id) %>
> > +            <!-- Line below generates receipt for purchasing this item, if possible -->
> > +            <% if logged_in? %>
>
> So currently there's actually no URL code in the sense that there's a URL
> I can enter which gives me the information I'm looking for.
> There's a receipt tracker tool that you submit your code to, and which
> returns your receipt if the code is correct.
>
> Each code is taken from a sample of 64^11, which makes them practically
> unique.
> 2) No site statistics are visible, or are discernible, though our current
> system.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Niupiao/niupiao-market/pull/80/files#r32906545>.
>